### PR TITLE
fix routing from internal.hail.is

### PR DESCRIPTION
**Pull request by cseed**
OK, this should fix routing from internal.hail.is.  The gateway routes internal.hail.is/ns/svc to router.ns with Host: svc.internal so the ns router can dispatch to the right server block off the Host.  We could dispatch off the URL, but that would mean the default and private namespaces dispatch different, doubling the complexity of the router NGINX configuration.  Change the host back for grafana and prometheus which generate a redirect otherwise.

The monitoring and gateway changes are deployed and everything seems to be working.

### CODE REVIEW

File: router/router.nginx.conf.in [hunk 1]
```diff
@@ -6,7 +6,7 @@ map $http_x_forwarded_proto $updated_scheme {
 }
 
 server {
-    server_name @domain@ www.@domain@;
+    server_name @domain@ www.* site.*;
```
Comment (jigold) [inline router/router.nginx.conf.in]: Do you still need domain here? If not, does it need to be taken out of the templating?

**Comment by jigold** [comment]
Although let me try again now that you changed the router code.

**Comment by jigold** [comment]
I get a 301 moved permanently now instead of the 404 from before.

File: router/router.nginx.conf.in (hunk 1 above)
Comment (cseed) [reply router/router.nginx.conf.in]: I think I do, because we route http://hail.is/ to this server block.
